### ISSUE TITLE
Homekit Schalt und Messsteckdose

### ISSUE THREAD

**Issue by DennisXK** [rdmtc/RedMatic#552]
Hey, ist es mögliche die Leistung der Messsteckdose HMIP-PSE in Homekit anzeigen zu lasen? Ich bekomme nur denn Schalter angezeigt

Grüße DennisXK

**Comment by blackflash73** [comment]
Ja, falls Daten in HomeKit vorhanden, kann man sie z.B. mit der EVE HomeKit-App anzeigen lassen. 
https://apps.apple.com/de/app/eve-f%C3%BCr-homekit/id917695792

Ich nutze dafür das CCU3-AddOn HAP-HomeMatic von THKL:
https://github.com/thkl/hap-homematic
(ist leider auch schon ewig nicht mehr aktualisiert worden, aber funktioniert - noch...)

VG BlackFlash

**Comment by blackflash73** [comment]
Bonus-Tipp:

Mit HomeSpy kann man sich alle HomeKit-Objekte anzeigen lassen und auch das Log lesen. ;)
https://apps.apple.com/de/app/homespy/id1520611941
